### PR TITLE
Display members list

### PR DESCRIPTION
**Pull request by byum1996**

### CODE REVIEW

File: src/members/email.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (justinyum98) [inline src/members/email.js:1]: not super big, but i would name the file "Email.js".

**Review by justinyum98 — COMMENTED** [review]
👍

File: src/members/Members.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    return (
+        <Member member={member} />
+    )
+}
+
+const Members = ({members}) => {
+
+    return (
+        <List>
+            { members.map((member) => {return renderMember(member)})}
```
Comment (sangyum) [inline src/members/Members.js:15]: You could simplify this further.

Remember, when an arrow function (aka anonymous function) has only `return` statement, you can remove curly braces and `return` keyword.

`{ members.map((member) => renderMember(member))`

**Review by sangyum — APPROVED** [review]
Minor nitipick but looks good

**Comment by byum1996** [comment]
> Minor nitipick but looks good

I actually did that right after I made the PR request but then forgot to commit it last night. I'll commit it right now

**Pull request merged**